doc/man/sphinx-build.rst

```diff
@@ -39,7 +39,8 @@ Options
    Extensions can add their own builders.
 
    .. important::
-      Sphinx only recognizes the ``-M`` option if it is used first.
+      Sphinx only recognizes the ``-M`` option if it is used first. The source
+      and output directories must also be passed before any options.
```

Comment:
So `-M` must come first, then source dir, then output dir, then all other options? Current phrasing is still not crystal clear IMO

Reply:
I wasn't sure the best either, what about:
```
Sphinx only recognizes the ``-M`` option if it is used first, along with
the source and output directories.
```

Reply:
Actually, wouldn't it be better to just include an example of what should be done and what should not ?

Reply:
+1 to a working example as a simple and effective way of squashing ambiguity

Reply:
Looks good to me

Reply:
Sweet. Updated in 02793bc0b489027d16edd82dc1fabb4ea0b313fd